percona-xtradb-cluster-8.0/Dockerfile.aarch64

```diff
@@ -124,6 +124,9 @@ RUN set -ex; \
     rm -rf /var/cache/dnf /var/cache/yum /var/lib/mysql /usr/lib64/mysql/plugin/debug /usr/sbin/mysqld-debug /usr/lib64/mecab /usr/lib64/mysql/mecab /usr/bin/myisam*; \
     rpm -ql percona-xtradb-cluster-client | egrep -v "mysql$|mysqldump$|mysqladmin$|mysqlbinlog$" | xargs rm -rf
 
+# fix PKG-842
+RUN sed -i '1756,1765 s/^/#/' /usr/bin/wsrep_sst_xtrabackup-v2
```

Comment:
This also fails:
```
if compare_versions "$XB_2x_VERSION" "<" "$XB_2x_REQUIRED_VERSION"; then
```

So should be:

`RUN sed -i '1756,1773 s/^/#/' /usr/bin/wsrep_sst_xtrabackup-v2`